vsintegration/src/FSharp.Editor/Navigation/NavigateToSearchService.fs

```diff
@@ -262,7 +264,7 @@ type internal FSharpNavigateToSearchService
         member __.SearchDocumentAsync(document, searchPattern, cancellationToken) : Task<ImmutableArray<INavigateToSearchResult>> =
             asyncMaybe {
                 let! options = projectInfoManager.TryGetOptionsForDocumentOrProject(document)
-                let! items = getCachedIndexedNavigableItems(document, options, cancellationToken) |> liftAsync
```

Comment:
generally F# code should not plumb a cancellation token explicitly, it is better to reacquire it at each Task --> Async conversion